content/en/serverless/installation/dotnet.md

```diff
@@ -13,32 +13,27 @@ further_reading:
   text: 'Submitting Custom Metrics from Serverless Applications'
 ---
 
-## Required setup
+## Prerequisites
 
-If not already configured:
-
-- Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics from AWS. 
-- Install the [Datadog Forwarder Lambda function][2], which is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs.
-
-After you have installed the [AWS integration][1] and the [Datadog Forwarder][2], follow these steps to instrument your application to send metrics, logs, and traces to Datadog.
+The [Datadog Forwarder Lambda function][1] is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs.
```

Comment:
So there are now two paths with universal instrumentation -- this one (with the Forwarder), and now a new one in beta using the Lambda Extension.

These are the instructions for using the Extension instead of the Forwarder:

1. Add the Datadog Lambda Extension layer for your Lambda function using the ARN in the following format:

// For x86 lambdas
arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:<EXTENSION_VERSION>
// For arm64 lambdas
arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension-ARM:<EXTENSION_VERSION>

2. Set the environment variable DD_API_KEY to your Datadog API key from API Management (https://app.datadoghq.com/organization-settings/api-keys).